packages/core/index.d.ts

```diff
@@ -51,6 +51,8 @@ declare module '@rjsf/core' {
         uiSchema?: UiSchema;
         validate?: (formData: T, errors: FormValidation) => FormValidation;
         widgets?: { [name: string]: Widget };
+        /** WARNING: This exists for internal react-jsonschema-form purposes only. Use ONLY if you know what you are doing */
```

Comment:
Can you add a one-to-two line description of what _internalFormWrapper is and how / when it should be used? For future maintainers of rjsf (and for us)

```suggestion
        /** WARNING: This exists for internal react-jsonschema-form purposes only. No guarantees of backwards compatibility. Use ONLY if you know what you are doing */
        /** The _internalFormWrapper prop is currently used by the material-ui and semantic-ui themes. One should pass a component that ...*/
```

Reply:
I am going to document it's usage in `Form` rather than in the types file if that is ok with you

Reply:
Sure